### PR TITLE
pyproject.toml: Work on the readability of ruff settings

### PR DESCRIPTION
**Pull request by cclauss**
# Description

pyproject.toml: Work on the readability of ruff settings

Please include a summary of the change and which issue is fixed. Please also
include relevant motivation and context. Your commit message should include
this information as well.

Fixes # (issue)

# Checklist:

- [x] I have added the relevant tests for this change.
- [ ] I have added an item to the Pending section of ``docs/changes.rst``.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -53,61 +53,45 @@ path = "debug_toolbar/__init__.py"
...
+  "B",      # flake8-bugbear
+  "C4",     # flake8-comprehensions
+  "C90",    # McCabe cyclomatic complexity
+  "DJ",     # flake8-django
+  "E",      # pycodestyle errors
+  "F",      # Pyflakes
+  "FBT",    # flake8-boolean-trap
+  "I",      # isort
+  "INT",    # flake8-gettext
+  # "N",    # pep8-naming
```
Comment (tim-schilling) [inline pyproject.toml:68]: We can probably just remove this line.

**Review by tim-schilling — APPROVED** [review]
I'm in favor of this change. It looks like the major differences are the line-length and complexity.

File: pyproject.toml [hunk 2]
```diff
@@ -53,61 +53,43 @@ path = "debug_toolbar/__init__.py"
...
 ]
 extend-ignore = [
-  # Allow zip() without strict=
-  "B905",
-  # No line length errors
-  "E501",
+  "B905",   # Allow zip() without strict=
 ]
 fix = true
+line-length = 124
```
[the remark this answers is not in the document]
Comment (cclauss) [reply pyproject.toml:79]: https://docs.astral.sh/ruff/faq/#is-ruff-compatible-with-black
> Note that Ruff and Black treat line-length enforcement a little differently...

[the remark this answers is not in the document]
Comment (matthiask) [reply pyproject.toml:79]: Yes, I use and like both. I don't see the value in adding a line length limitation; I think we should just let `black` format the code, and apart from that ignore all line length violations.

**Review by matthiask — COMMENTED** [review]
Thanks! I'm a bit indifferent (the previous ordering was based on the order here https://docs.astral.sh/ruff/rules/ , the current ordering is alphabetical according to the short name) but I appreciate your work.

**Comment by matthiask** [comment]
Thanks!

**Pull request merged**